include/aikido/control/ros/Conversions.hpp

```diff
@@ -28,6 +29,14 @@ std::unique_ptr<aikido::trajectory::Spline> toSplineJointTrajectory(
 trajectory_msgs::JointTrajectory toRosJointTrajectory(
   const aikido::trajectory::TrajectoryPtr& trajectory, double timestep);
 
+/// Converts Eigen VectorXd and joint names to JointState
+/// \param[in] goalPositions The required positions for the fingers
```

Comment:
Nit: This is not specific to finger joints! 😉 

Also, we should add "position" to the name of this function, since we may very well need equivalent functions for velocities and/or efforts. Perhaps `positionsToJointState`?